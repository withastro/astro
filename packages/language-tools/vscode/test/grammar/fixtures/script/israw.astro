<script is:raw>
	"hello"
</script>
