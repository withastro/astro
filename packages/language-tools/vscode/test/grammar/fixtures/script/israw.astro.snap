><script is:raw>
#^ source.astro meta.scope.tag.script.astro meta.raw.astro meta.tag.start.astro punctuation.definition.tag.begin.astro
# ^^^^^^ source.astro meta.scope.tag.script.astro meta.raw.astro meta.tag.start.astro entity.name.tag.astro
#       ^ source.astro meta.scope.tag.script.astro meta.raw.astro source.unknown meta.tag.start.astro
#        ^^^^^^ source.astro meta.scope.tag.script.astro meta.raw.astro source.unknown meta.tag.start.astro meta.attribute.is:raw.astro entity.other.attribute-name.astro
#              ^ source.astro meta.scope.tag.script.astro meta.raw.astro source.unknown meta.tag.start.astro punctuation.definition.tag.end.astro
>	"hello"
#^^^^^^^^^ source.astro meta.scope.tag.script.astro meta.raw.astro source.unknown
></script>
#^^ source.astro meta.scope.tag.script.astro meta.raw.astro meta.tag.end.astro punctuation.definition.tag.begin.astro
#  ^^^^^^ source.astro meta.scope.tag.script.astro meta.raw.astro meta.tag.end.astro entity.name.tag.astro
#        ^ source.astro meta.scope.tag.script.astro meta.raw.astro meta.tag.end.astro punctuation.definition.tag.end.astro
>